tools/wpt/expectation.json

```diff
@@ -171,7 +171,49 @@
+      "encodeInto() into SharedArrayBuffer with 𝌆 and destination length 4, offset 0, filler 128",
+      "encodeInto() into SharedArrayBuffer with 𝌆 and destination length 4, offset 4, filler 128",
+      "encodeInto() into SharedArrayBuffer with 𝌆 and destination length 4, offset 0, filler random",
+      "encodeInto() into SharedArrayBuffer with 𝌆 and destination length 4, offset 4, filler random",
+      "encodeInto() into SharedArrayBuffer with 𝌆A and destination length 3, offset 0, filler 0",
+      "encodeInto() into SharedArrayBuffer with 𝌆A and destination length 3, offset 4, filler 0",
+      "encodeInto() into SharedArrayBuffer with 𝌆A and destination length 3, offset 0, filler 128",
+      "encodeInto() into SharedArrayBuffer with 𝌆A and destination length 3, offset 4, filler 128",
+      "encodeInto() into SharedArrayBuffer with 𝌆A and destination length 3, offset 0, filler random",
+      "encodeInto() into SharedArrayBuffer with 𝌆A and destination length 3, offset 4, filler random",
+      "encodeInto() into SharedArrayBuffer with \ud834A\udf06A¥Hi and destination length 10, offset 0, filler 0",
+      "encodeInto() into SharedArrayBuffer with \ud834A\udf06A¥Hi and destination length 10, offset 4, filler 0",
+      "encodeInto() into SharedArrayBuffer with \ud834A\udf06A¥Hi and destination length 10, offset 0, filler 128",
+      "encodeInto() into SharedArrayBuffer with \ud834A\udf06A¥Hi and destination length 10, offset 4, filler 128",
+      "encodeInto() into SharedArrayBuffer with \ud834A\udf06A¥Hi and destination length 10, offset 0, filler random",
+      "encodeInto() into SharedArrayBuffer with \ud834A\udf06A¥Hi and destination length 10, offset 4, filler random",
+      "encodeInto() into SharedArrayBuffer with A\udf06 and destination length 4, offset 0, filler 0",
+      "encodeInto() into SharedArrayBuffer with A\udf06 and destination length 4, offset 4, filler 0",
+      "encodeInto() into SharedArrayBuffer with A\udf06 and destination length 4, offset 0, filler 128",
+      "encodeInto() into SharedArrayBuffer with A\udf06 and destination length 4, offset 4, filler 128",
+      "encodeInto() into SharedArrayBuffer with A\udf06 and destination length 4, offset 0, filler random",
+      "encodeInto() into SharedArrayBuffer with A\udf06 and destination length 4, offset 4, filler random",
+      "encodeInto() into SharedArrayBuffer with ¥¥ and destination length 4, offset 0, filler 0",
+      "encodeInto() into SharedArrayBuffer with ¥¥ and destination length 4, offset 4, filler 0",
+      "encodeInto() into SharedArrayBuffer with ¥¥ and destination length 4, offset 0, filler 128",
+      "encodeInto() into SharedArrayBuffer with ¥¥ and destination length 4, offset 4, filler 128",
+      "encodeInto() into SharedArrayBuffer with ¥¥ and destination length 4, offset 0, filler random",
+      "encodeInto() into SharedArrayBuffer with ¥¥ and destination length 4, offset 4, filler random",
```

Comment:
SharedArrayBuffer serialization was disabled in https://github.com/denoland/serde_v8/pull/4